### PR TITLE
After demo updates

### PR DESCRIPTION
**Pull request by AbdallahAbuHashem**
Changes here are things that came up as a result of the demo

1- List sources when printing outdated
2- Change the resource for outdated to avoid getting unlisted versions
3- Print the short name of a package instead of the long name

### CODE REVIEW

File: src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx [hunk 1]
```diff
@@ -570,4 +570,7 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="ListPkg_ProjectUpdatesHeaderLog" xml:space="preserve">
     <value>Project `{0}` has the following updates to its packages</value>
   </data>
+  <data name="ListPkg_SourcesUsedDescription" xml:space="preserve">
+    <value>The following sources will be used for outdated:</value>
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx]: nit: `The following sources were used` or `Feeds used`. 

That way when we add deprecated we don't have to change the text :) 

https://github.com/NuGet/NuGet.Client/blob/f198ea2d3a32ce91fd11b0104f81021a3f5054c5/src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreSummary.cs#L117-L125 is the restore code and it uses `Feeds used`

File: src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs [hunk 2]
```diff
@@ -326,9 +335,9 @@ private UpdateLevel GetUpdateLevel(NuGetVersion resolvedVersion, NuGetVersion la
             Dictionary<string, IList<NuGetVersion>> packagesVersionsDict)
         {
             var sourceRepository = Repository.CreateSource(providers, packageSource, FeedType.Undefined);
-            var dependencyInfoResource = await sourceRepository.GetResourceAsync<FindPackageByIdResource>(listPackageArgs.CancellationToken);
+            var dependencyInfoResource = await sourceRepository.GetResourceAsync<MetadataResource>(listPackageArgs.CancellationToken);
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs:338]: I'm not sure I follow this change. 
Why do we do it now? What are the benefits?

File: src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ProjectPackagesPrintUtility.cs [hunk 3]
```diff
@@ -211,5 +211,13 @@ internal static string[] BuildTableHeaders(bool printingTransitive, bool outdate
 
             return result.ToArray();
         }
+
+        internal static void PrintSources(IEnumerable<PackageSource> packageSources)
+        {
+            foreach (var source in packageSources)
+            {
+                Console.WriteLine(source.Source);
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ProjectPackagesPrintUtility.cs]: I'd imagine you want some indentation there similar to what we do in restore.

**Review by nkolev92 — COMMENTED** [review]
Couple of small things. 

Grab me offline if needed.

File: src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs (hunk 2 above)
Comment (AbdallahAbuHashem) [reply src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs:338]: Previously it was getting all of the versions, but we only want the listed ones. The resource that I added does that

Comment (nkolev92) [reply src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs:338]: I see. 
I'd imagine there's a listed/unlisted field on the metadata there.

File: src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ProjectPackagesPrintUtility.cs [hunk 4]
```diff
@@ -211,5 +211,13 @@ internal static string[] BuildTableHeaders(bool printingTransitive, bool outdate
 
             return result.ToArray();
         }
+
+        internal static void PrintSources(IEnumerable<PackageSource> packageSources)
+        {
+            foreach (var source in packageSources)
+            {
+                Console.WriteLine("   " + source.Source);
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ProjectPackagesPrintUtility.cs:219]: just fyi, you don't need to change it.
$"    {source.Source}" - string interpolation is usually more efficient than string concatenation.

**Review by nkolev92 — APPROVED** [review]
LGTM

**Pull request merged**